### DISCUSSION TITLE
automatic e-mail notification when speakers receive new feedback

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretalx/pretalx#1573] [Ideas] [answered]
I wondered if the feedback function was not working at our Pretalx instance because I assumed that the speakers get an email when feedback arrives.

Apparently they can "only" see this in their account, so I wanted to ask if it would be possible to add such an automatic email function.

A "There is new feedback on Talk XY in your profile" would be enough, but if the few feedback lines could be included in the email, that would be even nicer.

Thank you very much for all your work, we are very happy that we can use Pretalx as it is!

**Comment by rixx** [comment, marked as answer]
Sorry, I'm not currently planning on adding email notifications like that. The feedback site is currently a simple open web form, which means it's an obvious target for spammers. Adding email notifications would mean also adding opt-in and opt-out for speakers, which would include a completely new email notification management page, opt-out links that work when not logged in (so saving opt-out uuids somewhere) etc. Just not worth it for a rather niche feature that many events currently don't use.

If anybody wants to build all of the above, I'd of course be happy to guide and review a PR.

**Reply by navlisilvan** [reply]
I see the point(s), thx.
